presto-main/src/test/java/com/facebook/presto/sql/planner/TestPredicatePushdown.java

```diff
@@ -251,4 +259,45 @@ public void testFilteredSelectFromPartitionedTable()
                         values("orderstatus")),
                 allOptimizers);
     }
+
+    @Test
+    public void testInformationSchemaColumnsTable()
```

Comment:
This seems like the wrong place for the test: this class is for engine tests, but the code change and behavior is limited to the connector. I think we could move the test into the connector package.

Reply:
Moved.